oca_port/migrate_addon.py

```diff
@@ -240,3 +242,10 @@ def _print_tips(self, blacklisted=False):
             new_pr_url=new_pr_url,
         )
         print(tips)
+
+    def _apply_code_pattern(self):
+        print("Apply code pattern...")
+        subprocess.run(
+            f"odoo-module-migrate -i {self.app.source_version} -t {self.app.target_version} --modules {self.app.addon} --directory {self.app.repo_path} ",
+            shell=True,
+        )
```

Comment:
I would make `odoo-module-migrator` dependency optional, if it's found in the runtime then `oca-port` can leverage it.
I found out that `odoo-module-migrator` is available only for Python 3.10+, and I prefer to stay compatible with older Python releases if possible.